### PR TITLE
tailwind Implementation and create post page

### PR DESCRIPTION
**Pull request by Rhondoid**

### CODE REVIEW

File: controllers/htmlRoutes/createPostPageRoutes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const { Post, User } = require('../../models');
+const withAuth = require('../../utils/auth')
+
+router.get('/', withAuth, async (req, res) => {
+
```
Comment (Jacee94) [inline controllers/htmlRoutes/createPostPageRoutes.js]: Can we remove this extra whitespace? Let's try to keep it pretty :)

Comment (Jacee94) [inline controllers/htmlRoutes/createPostPageRoutes.js]:
```suggestion
```

File: public/css/style.css [hunk 2]
```diff
@@ -8,4 +8,12 @@
 
 #title {
     @apply text-lg font-bold  pl-1.5;
-}
\ No newline at end of file
+}
+.form-input {
+    @apply border-2 border-accentgreen border-solid;
+    @apply text-sm  text-accentgreen pl-1;
+
```
Comment (Jacee94) [inline public/css/style.css]:
```suggestion
```

**Review by Jacee94 — CHANGES_REQUESTED** [review]

File: public/css/tailwind.css [hunk 3]
```diff
@@ -640,7 +644,26 @@ video {
...
+  color: rgb(3 43 19 / var(--tw-text-opacity));
+}
+
+.btn-primary {
+  border-width: 2px;
+  border-style: solid;
+  --tw-border-opacity: 1;
+  border-color: rgb(3 43 19 / var(--tw-border-opacity));
+}
+
```
Comment (Jacee94) [inline public/css/tailwind.css:665]: If this is all our own custom styling shouldn't we keep it out of the tailwind.css and put it in our own css file?

**Review by Jacee94 — CHANGES_REQUESTED** [review]

**Review by bikeller104 — APPROVED** [review]

**Review by Jacee94 — APPROVED** [review]

**Pull request merged**